tree/tree/src/TTree.cxx

```diff
@@ -809,7 +810,6 @@ TTree::TTree(const char* name, const char* title, Int_t splitlevel /* = 99 */)
    // FIXME: This is very annoying behaviour, we should
    //        be able to choose to not do this like we
    //        can with a histogram.
-   fDirectory = gDirectory;
```

Comment:
Should the FIXME comment above have been removed? Your commit fixes the "annoying behavior", no?